scripts/provisioning/s3_haproxy_config.py

```diff
@@ -358,11 +368,12 @@ def configure_haproxy_k8(self):
     pem_dir = os.path.dirname(sslcertpath)
```

Comment:
Need to use correct ssl cert path in haproxy config above, line 361.  Not sure if this cert is the same as sslcertpath or not.

```
server s3authserver-instance1 0.0.0.0:%s #check ssl verify required ca-file /etc/ssl/stx-s3/s3auth/s3authserver.crt   
```

Reply:
In line 361 - `server s3authserver-instance1 0.0.0.0:%s` we are adding port number, not ssl certificate path.